contrib/google.golang.org/grpc/option.go

```diff
@@ -1,14 +1,25 @@
 package grpc
 
+import (
+	"google.golang.org/grpc/codes"
+)
+
 // Option specifies a configuration option for the grpc package. Not all options apply
 // to all instrumented structures.
 type Option = InterceptorOption
 
 type config struct {
-	serviceName                           string
-	traceStreamCalls, traceStreamMessages bool
-	noDebugStack                          bool
-	analyticsRate                         float64
+	serviceName string
+
+	// codes.Code values will be used to index into this slice.
+	// Given a codes.Code "c" the expression nonErrorCodes[c]
+	// yields whether this code is not to be regarded as an error.
+	nonErrorCodes []bool
```

Comment:
Why not a `map[codes.Code]bool`? Seems more obvious.

Reply:
We'd like to favor readability here, so please go ahead and use the hashmap and remove `setNonErrorCodes` and `isNonErrorCode`. Let's keep it simple